src/dependency-registry.ts

```diff
@@ -367,7 +415,7 @@ export function handleSupportedCoreContractRemoved(
   event: SupportedCoreContractRemoved
 ): void {
   const coreContract = Contract.load(
```

Comment:
unrelated to this PR - probably worth a check that `coreContract.dependencyRegistry == event.address` before setting it to null? (I know we only expect one dependency registry, but who knows...)

Reply:
Agreed! https://app.asana.com/0/1201568815538912/1206099136189468/f